toolkit/cli/smart-contracts-commands/src/governance.rs

```diff
@@ -58,9 +58,15 @@ impl InitGovernanceCmd {
 pub struct UpdateGovernanceCmd {
 	#[clap(flatten)]
 	common_arguments: crate::CommonArguments,
-	/// Governance authority hash to be set.
-	#[arg(long, short = 'g')]
-	new_governance_authority: MainchainKeyHash,
+	/// Old governance authority hash
```

Comment:
Wouldn't "current" fit better than "old"?
IMO "Old/Current governance authorities" or "Old/Current governance authorities keys hashes".
With addition of ", hex encoded, space delimited, order does not matter"